README.md

```diff
@@ -18,14 +18,27 @@ fly as soon as a certain number of new features and fixes have been made.
 PyCayenneLPP does not have any external dependencies, but only uses builtin
 functions and types of Python 3. At least Python in version 3.4 is required.
 
-### Prerequisites
+### Python 3 Prerequisites
 
 The PyCayenneLPP package is available via PyPi using `pip`. To install it run:
 
 ```Shell
 pip3 install pycayennelpp
 ```
 
+### MicroPython Prerequisites
+
+MicroPython does not include the libraries `base64` and `logging` per default.
+While the latter rather optional for embedded devices, the former is essential.
+It can be installed from the
+[micropython-lib](https://github.com/micropython/micropython-lib/tree/master/base64)
+project via tools like [ampy](https://github.com/adafruit/ampy).
+
+    git clone https://github.com/micropython/micropython-lib.git
```

Comment:
according to [codacy](https://app.codacy.com/app/smlng/pycayennelpp/pullRequest?prid=3145595) this should rather be enclosed with ``` before and after the code/commands.

Though indention by 4 spaces is typically considered code in markdown too, anyway: please adapt, THX!

Reply:
finally all done?

Reply:
yep, looks good, and codacy is happy too 😄 Thanks for contributing!

I'll prepare a new release announcing MicroPython support 🎉  within the next few days.